message_handler.py

```diff
@@ -46,6 +48,23 @@ def handle_message_invalid():
     '''
     return 'I don\'t understand your accent. Try !help'
 
+def handle_message_crate_time():
+    # Get current UTC time (UTC is common for world).
+    current_utc_time = datetime.datetime.utcnow()
+
+    # Turn UTC time to correct time zone (-7).
+    current_timezone_time = current_utc_time + datetime.timedelta(hours=-7)
+    
+    # Find the difference in the corrected time and Sunday at 6PM.
+    next_crate_time = current_timezone_time.replace(hour=18, minute=0, second=0)
```

Comment:
Or maybe better to just do utc here

Reply:
If datetime supports UTC only, everything needs to be converted to UTC. No need to convert to PST

Reply:
Also need to explain the negative time thing.. comment is ambiguous